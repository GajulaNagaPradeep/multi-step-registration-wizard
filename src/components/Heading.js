import React from "react";
import Progress from "./Progress";

const Heading = () => (
  <div>
    <h1>Multi Step Registration Widget</h1>
    <Progress />
  </div>
);

export default Heading;
