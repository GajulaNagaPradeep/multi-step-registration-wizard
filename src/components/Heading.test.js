import React from "react";
import { shallow } from "enzyme";
import Heading from "./Heading";

const title = "Test Title";
let wrapped = shallow(<Heading>{title}</Heading>);
describe("Heading", () => {
  it("should be rendered correctly", () => {
    expect(wrapped).toMatchSnapshot();
  });
});
