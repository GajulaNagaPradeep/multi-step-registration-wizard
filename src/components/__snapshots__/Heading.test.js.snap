// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should be rendered correctly 1`] = `
<div>
  <h1>
    Multi Step Registration Widget
  </h1>
  <withRouter(Progress) />
</div>
`;
